scripts/tests/versioner_hashes.yaml

```diff
@@ -863,3 +863,34 @@ fd5f7bc651f0d6a8e2d23e5d4c2c2dadca58aa0b: #29978
   wasp:
     full_version: 2025.02.25_0
     hash: 46ea1c0
+9c3c8d833ce9c5d8fbb748dc6b870a075f4b2827:
```

Comment:
Noted that this is missing a ticket number comment while rebasing #29984. Will add in a commit in that PR for completeness.

Reply:
This must be something missing in `--build-template`

Reply:
I thought the specific ticket number was something we should be adding manually after copy-pasting the `versioner.py --summary` output into `versioner_hashes.yaml`.

Reply:
You're right. I never scrolled that far in the wiki instructions 😆 I do think that this third step should also be somehow automated and included in `--build-template`